care/facility/api/viewsets/patient.py

```diff
@@ -29,6 +31,10 @@
 from care.facility.models.patient_icmr import PatientIcmr
 from care.users.models import User
 
+from care.facility.tasks.patient.discharge_report import generate_discharge_report
+
+from config.celery_app import app
```

Comment:
Codacy found an issue: [Imports from package config are not grouped](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5811718)